modules/andes-core/client/src/main/java/org/wso2/andes/client/protocol/AMQProtocolHandler.java

```diff
@@ -575,7 +575,7 @@ public void run()
         });
         if (PROTOCOL_DEBUG)
         {
-            _protocolLogger.debug(String.format("SEND: [%s] %s", this, frame));
+            _protocolLogger.info(String.format("SEND: [%s] %s", this, frame));
```

Comment:
logic is as follows :

bool PROTOCOL_DEBUG = sysprop(amqj.protocol.logging.level) != null.

its the same value as the systemprop.